src/main/java/com/taf/business/service/LoginService.java

```diff
@@ -0,0 +1,22 @@
+package com.taf.business.service;
+
+import com.taf.business.pages.LoginPage;
+
+import static com.codeborne.selenide.Selenide.open;
+import static com.codeborne.selenide.Selenide.page;
+
+public class LoginService {
+
+    private final LoginPage loginPage = page(LoginPage.class);
+
+    private void openLoginPage() {
+        open("/");
+    }
+
+    public void loginWithUser() {
+        openLoginPage();
```

Comment:
from my point of view it's not needed to create wrapper for `open("/")` selenide method, you can use it directly here

Reply:
Right. It was removed.